src/HeaderSort/components/HeaderSortContainer.ts

```diff
@@ -34,7 +34,7 @@ interface FormState {
 
 export default class HeaderSortContainer extends Component<ContainerProps, ContainerState> {
     private dataSourceHelper?: DataSourceHelper;
-    private widgetDom: HTMLElement | null = null;
+    private widgetDom: Element | null = null;
```

Comment:
I think the function "connectToListView" should also be changed over here.

Reply:
No validation based on stated